internal/antewrapper/msg_fees_decorator.go

```diff
@@ -108,6 +117,13 @@ func (afd MsgFeesDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool
 	return next(ctx, tx, simulate)
 }
 
+//   The chain-id check is exclusively for not breaking all existing sim tests which freak out when denom is anything other than stake.
+//   and some network tests won't work without a chain id being set(but they also setup everything with stake denom) so `simapp-unit-testing` chain id is skipped also.
+//   This only needs to work to pio-testnet and pio-mainnet, so this is safe.
+func chainIdsToIgnoreForFloorGasPriceCheck(ctx sdk.Context) bool {
```

Comment:
NIT: Can we rename this to something else?  At first look, I can't tell if it should return `true` or `false` by the name.  Maybe something like `shouldIgnoreFloorGasPriceCheck`?

Reply:
renamed